chat-plugins/info.js

```diff
@@ -1337,6 +1337,15 @@ exports.commands = {
 		if (showAll || target === 'tiering' || target === 'tiers' || target === 'tier') {
 			buffer.push("<a href=\"https://www.smogon.com/ingame/battle/tiering-faq\">Tiering FAQ</a>");
 		}
+		if (showAll || toId(target).startsWith('lostpass') || target === 'pwreset') {
+			buffer.push("PM a member of Global Staff (@ in the Help room) to facilitate password resets.");
+		}
+		if (showAll || toId(target) === 'lostteams' || toId(target) === 'deletedteams') {
+			buffer.push("Have you cleared your cookies? This deletes your teams. You may  have done this accidentally when clearing your history, or by using  private browsing. Always back up your teams to a pastebin!");
```

Comment:
I'd recommend backing up locally over using a pastebin, unless the user's on a tablet or something and doesn't have much choice